volatility3/framework/symbols/linux/__init__.py

```diff
@@ -367,9 +368,8 @@ def files_descriptors_for_process(
                 yield fd_num, filp, full_path
 
     @classmethod
-    @Deprecation.deprecated_method(
-        replacement=linux_utilities_modules.Modules.mask_mods_list,
-        replacement_version=(1, 0, 0),
+    @Deprecation.method_being_removed(
+        "This method is not needed. The correct API for mapping kernel pointers to modules is `linux_utilities_modules.Modules.run_module_scanners`"
```

Comment:
The message should be something a little softer but more direct.  "This method is no longer recommended and will be removed from the framework in a future release.  Code using this method should be adapted to make use of `linux_utilities_modules.Modules.run_module_scanners`"

Not needed implies it's fine to keep using it, and "correct API" sounds like you're calling this one wrong, which is wasn't right up until we deprecated it (and it'll actually still work afterwards just fine for a while).